virtcontainers/qemu.go

```diff
@@ -658,6 +660,14 @@ func (q *qemu) stopSandbox() error {
 
 	err = q.qmpMonitorCh.qmp.ExecuteQuit(q.qmpMonitorCh.ctx)
```

Comment:
@Ace-Tang I am surprised that the earlier qmp.Setup() succeeds, so looks like for the qemu exits between qmpSetup and  qmp.ExecuteQuit() call.